cmd/run.go

```diff
@@ -12,11 +12,11 @@ import (
 
 	"github.com/argoproj-labs/argocd-image-updater/pkg/argocd"
 	"github.com/argoproj-labs/argocd-image-updater/pkg/common"
-	"github.com/argoproj-labs/argocd-image-updater/pkg/env"
-	"github.com/argoproj-labs/argocd-image-updater/pkg/health"
 	"github.com/argoproj-labs/argocd-image-updater/pkg/metrics"
 	"github.com/argoproj-labs/argocd-image-updater/pkg/registry"
 	"github.com/argoproj-labs/argocd-image-updater/pkg/version"
+	"github.com/argoproj-labs/argocd-image-updater/registry-scanner/pkg/env"
+	"github.com/argoproj-labs/argocd-image-updater/registry-scanner/pkg/health"
 	"github.com/argoproj-labs/argocd-image-updater/registry-scanner/pkg/log"
```

Comment:
Is the health server referenced by registry-scanner for it to be relocated?

Reply:
It is not. Do you feel we should keep it in image-updater itself?

Reply:
Yeah, since it's checks the health of argocd-image-updater, and it won't be able to run within registry-scanner. We can address that in a separate PR.